mopidy_musicbox_webclient/static/js/gui.js

```diff
@@ -308,43 +308,35 @@ $(document).bind("pageinit", function() {
 /**************
  * gui stuff  *
  **************/
-function enterFullscreen() {
+function toggleFullscreen() {
     if (isMobileSafari) { alert ("To get this app in Full Screen, you have to add it to your home-screen using the Share button."); exit(); }
-    var elem = document.querySelector("#page");
-    elem.onwebkitfullscreenchange = onFullScreenEnter;
-    elem.onmozfullscreenchange = onFullScreenEnter;
-    elem.onfullscreenchange = onFullScreenEnter;
-    if (elem.webkitRequestFullscreen) {
-        elem.webkitRequestFullscreen(Element.ALLOW_KEYBOARD_INPUT);
+    if (!document.fullscreenElement &&    // alternative standard method
```

Comment:
I couldn't find any direct support for fullscreen mode in jQuery. There are a bunch of jQuery plugins that does the same thing, but I'm a little reluctant to rely on those as a rule - maintenance for many of them appear to be spotty at best.